velox/expression/tests/ExpressionRunner.cpp

```diff
@@ -27,40 +29,113 @@
 
 namespace facebook::velox::test {
 
-using namespace facebook::velox;
+namespace {
+std::vector<core::TypedExprPtr> parseSql(
+    const std::string& sql,
+    const TypePtr& inputType,
+    memory::MemoryPool* pool) {
+  auto exprs = parse::parseMultipleExpressions(sql, {});
+
+  std::vector<core::TypedExprPtr> typedExprs;
+  typedExprs.reserve(exprs.size());
+  for (const auto& expr : exprs) {
+    typedExprs.push_back(core::Expressions::inferTypes(expr, inputType, pool));
+  }
+  return typedExprs;
+}
+
+RowVectorPtr createRowVector(
+    const std::vector<VectorPtr>& vectors,
+    vector_size_t size,
+    memory::MemoryPool* pool) {
+  auto n = vectors.size();
+
+  std::vector<std::string> names;
+  names.reserve(n);
+  std::vector<TypePtr> types;
+  types.reserve(n);
+  for (auto i = 0; i < n; ++i) {
+    names.push_back(fmt::format("_col{}", i));
```

Comment:
nit: i wonder if its possible to print the expression->sql() as the column name as the result for each expression